src/main/java/com/greetingcard/security/DefaultSecurityService.java

```
@@ -60,7 +67,26 @@ public void save(User user) {
     }
 
     @Override
-    public void update(User user) {
+    public void update(User user, MultipartFile file) {
+        String profileFile = "profile";
+        if (!file.isEmpty()) {
+            String uuidFile = UUID.randomUUID().toString();
+            String fileName = uuidFile + "." + file.getOriginalFilename();
+            try {
+                Files.createDirectories(Path.of(pathToFile,profileFile));
+                file.transferTo(new File(String.valueOf(Path.of(pathToFile,profileFile,fileName))));
+            } catch (IOException e) {
+                log.error("Can not save new photo: {}", fileName);
+                throw new RuntimeException("Can not save new photo",e);
+            }
```

Why do you use RuntimeException?

---

I need to throw an unchecked exception. And RuntimeException good candidate